kodilla-testing2/src/main/java/com/kodilla/testing2/facebook/FacebookTestingApp.java

```diff
@@ -0,0 +1,42 @@
+package com.kodilla.testing2.facebook;
+
+import com.kodilla.testing2.config.WebDriverConfig;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.Select;
+
+import java.util.concurrent.TimeUnit;
+
+public class FacebookTestingApp {
+
+    public static final String XPATH_SELECT_DAY = "//span[contains(@class, \"_5k_4\")]/span/select[1]";
```

Comment:
Rozmawialiśmy, żeby wszędzie gdzie się da, odnosić się do `id` - to pewniejsze rozwiązanie niż opieranie się o klasę i/lub kolejność elementu, gdyż jest mniejsze prawdopodobieństwo, że się zmieni.